### PR TITLE
`Scoop.summary`: summary.state should be an array

### PR DESCRIPTION
**Pull request by matteocargnelutti**
See: https://github.com/harvard-lil/perma-2023/issues/4#issuecomment-1536761274

Thanks @rebeccacremona

### CODE REVIEW

**Comment by matteocargnelutti** [comment]
This change is _somewhat_ breaking and will require the release of `0.3.0` as opposed to `0.2.8`.

File: Scoop.js [hunk 1]
```diff
@@ -1367,7 +1367,7 @@ export class Scoop {
   /**
    * @typedef {Object} ScoopCaptureSummary
    * @property {int} state
-   * @property {object} states - Possible values of Scoop.state
+   * @property {string[]} states - Indexed Scoop.states values.
```
Comment (rebeccacremona) [inline Scoop.js:1370]: Is it worth saying "Zero-indexed," or is that obvious>

**Review by rebeccacremona — APPROVED** [review]
Huzzah!

File: Scoop.js (hunk 1 above)
Comment (matteocargnelutti) [reply Scoop.js]: Fixed :) !

**Pull request merged**